config/environments/test.rb

```diff
@@ -4,26 +4,29 @@
 Argo::Application.configure do
   # Settings specified here will take precedence over those in config/application.rb
 
-  config.assets.compress = Settings.APPLICATION.ASSETS.COMPRESS
+  # The test environment is used exclusively to run your application's
```

Comment:
i'd be inclined to put this at the top of the file, since it's a general comment on the env and not specific to the `cache_classes` setting.